package.json

```diff
@@ -46,7 +46,8 @@
       "<rootDir>/src/jest.config.js"
     ],
     "collectCoverageFrom": [
-      "./src/**"
+      "<rootDir>/src/**/*.{js,jsx,json}",
```

Comment:
right. but JSON to be excluded or rather - not included

Reply:
@skyboyer Fixed